src/qt/applicationbase/inviwoapplicationqt.cpp

```diff
@@ -329,12 +329,22 @@ bool InviwoApplicationQt::notify(QObject* receiver, QEvent* e) {
         }
         case QEvent::TouchEnd: {
             auto te = static_cast<QTouchEvent*>(e);
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+            if (util::all_of(te->points(), [](const QTouchEvent::TouchPoint& tp) {
+                    return tp.state() == QEventPoint::Released;
```

Comment:
Only this line that differ

Reply:
te->points() vs te->touchPoints() and QEventPoint::Released vs Qt::TouchPointReleased have the ifdef on the whole block makes it much more readable